### ISSUE TITLE
Documentation for the `-ca` flag.

### ISSUE THREAD

**Issue by mqsoh** [mholt/caddy#850]
tl;dr I think you should mention the default URL or link to Let's Encrypt docs.

I wanted to switch to the Let's Encrypt staging server. These are the hoops I jumped through:

The [Automatic HTTPs page](https://caddyserver.com/docs/automatic-https) mentions a staging URL for Let's Encrypt. I didn't know where to find the staging URL, so I checked their service status. I tried the staging domain and found out I needed to add a protocol (from the error message) and added `https`. Then I found out that the URL was wrong. Finally I found [the line in the code that assigned the default value](https://github.com/mholt/caddy/blob/1f29c52151a4f040104eca0d3b9cd283c108f576/main.go#L25) and discovered that I needed a `/directory` on the end.

**Comment by mholt** [comment]
You're definitely right, it's not obvious right now. I hesitate to hard-code these URLs into the docs because Caddy works with any ACME-capable CA, and I also don't want to have to change the URLs in the docs if Let's Encrypt changes them. Since you opened this issue, others who have the question will be able to find the answer.

Fortunately, Let's Encrypt has [instructions](https://acme-staging.api.letsencrypt.org/) on which endpoint to use as the starting point, which specifically links to the directory.
